src/main/scala/nl/knaw/dans/easy/multideposit/package.scala

```diff
@@ -124,7 +124,6 @@ package object multideposit {
       * Writes a CharSequence to a file creating the file if it does not exist using the default encoding for the VM.
       *
       * @param data the content to write to the file
-      * @throws IOException in case of an I/O error
```

Comment:
I really like to have these `@throws` here to remind us all that these methods can throw. On the other hand, we could also choose to go the other route of wrapping it in `Try`, but that would require some more effort (maybe make a JIRA issue for that?).